README.md

```diff
@@ -45,6 +45,17 @@ Phone (one-tap):
 Phone (international):
 - https://VMware.zoom.us/u/asnmqyfmb
 
+#### Agenda
+
+A typical meeting might have the following agenda:
+
+10:00 - 10:15 - introductions, release updates, housekeeping
+10:15 - 10:30 - scheduled topic #1
+10:30 - 10:45 - scheduled topic #2
+10:45 - 11:00 - scheduled topic #3
+
+Topics should be added to the agenda in the public [Google Doc](https://docs.google.com/document/d/1CwixRknj3gAXbi-fTI1-chwtMfy6G3GhdH17aEUjhK0/edit) the day prior to the meeting.
```

Comment:
Is it someone's job to decide what the topics are if there are more than 3? Is it first come first serve?

Reply:
I think in practice it might be first come first serve, but we could leave it up to the discretion of the organizers.

Looking at https://github.com/buildpacks/community/pull/107, do you think review of open RFCs will move to that meeting? That might free up some time in WG to cover more topics or just leave space for ad-hoc discussion.

Reply:
yeah, that's my intent but I'm open to feedback.

> I think in practice it might be first come first serve, but we could leave it up to the discretion of the organizers.
It's probably worth being clear on how that's done.

Stuff gets added to Future Topics and the organizer picks 3 topics the day before.

> Looking at #107, do you think review of open RFCs will move to that meeting? That might free up some time in WG to cover more topics or just leave space for ad-hoc discussion.
That's the plan, but I'm open to feedback.

Reply:
```suggestion
Topics should be added to the agenda in the public [Google Doc](https://docs.google.com/document/d/1CwixRknj3gAXbi-fTI1-chwtMfy6G3GhdH17aEUjhK0/edit) the day prior to the meeting. The final list of topics will be finalized by the meeting organizers. If any scheduled topics are covered in less than 15 minutes, additional topics may be covered. 
```

Reply:
@natalieparellano, I like your suggestion but I think we should also remove the times on lines 52-55, otherwise it can be confusing when we're going to cover the additional topics (between the 3 main topics or at the end).

Maybe, a day before the meeting:
- Topics should be added to "future topics" by the morning (for example 9am EST).
- Organizers should create 3 lists for the meeting by noon (EST):
  1. introductions, release updates, housekeeping (up to 15 minutes in total)
  2. main topics (up to 15 minutes each)
  3. additional topics (if we have time)

In this way, community members will have ~0.5 day to review the agenda before the meeting.